warden/warden_daemon/src/cli.rs

```diff
@@ -16,6 +17,12 @@ pub struct Cli {
     pub unix_sock_path: PathBuf,
     #[arg(short, long)]
     pub warden_workdir_path: PathBuf,
+    #[arg(short, long)]
+    pub dhcp_exec_path: PathBuf,
     #[arg(short = 't', long, default_value_t = 60)]
     pub realm_connection_wait_time_secs: u64,
+    #[arg(short, long, default_value_t = String::from("virtbWarden"))]
+    pub bridge_name: String,
+    #[arg(short='i', long, default_value_t=IpNet::V4(Ipv4Net::new(Ipv4Addr::new(192, 168, 100, 0), 24).expect("This should be valid IP4 addr.")))]
```

Comment:
If I recall correctly this is called "network address" as you are later adding 1 to calculate the exact interface address.

Reply:
Done